You can no longer print infinite ammunition.

DaxDupont opened the pull request:
[Changelogs]: 

:cl: Dax Dupont
balance: After successful lobbying by the Space Ancap Federation ammo is no longer free. Material redemption cost is now determined by the amount of bullets left.
balance: .38 ammo is now cheaper to produce. 
balance: Casings now give a little bit of metal. mostly for immersion sake.
/:cl:

[why]: After seeing ye olde print infinite ammo murderboning I figured this would be a change that people might get behind. 
Things like the .357 are quite powerful, and being able to print and shoot infinite amounts of ammo by reclaiming the full amount on an empty mag seems obscene.
Might tweak the base cost if someone has input.

DaxDupont committed:
You can no longer print infinite ammunition.

vuonojenmustaturska commented on code/modules/projectiles/boxes_magazines/_box_magazine.dm:
```diff
@@ -21,9 +21,19 @@
 	var/multiload = 1
 	var/start_empty = 0
+	var/list/bullet_cost = list() 
+	var/list/base_cost = list() // override this one as well if you override bullet_cost
```
initialize these lists as null, otherwise you're throwing memory away

BeeSting12 commented:
Nerf to the detective. Probably healthy for the game but 👎 because i like abusing this

DaxDupont committed:
henk

lazy

anus

Reduces .38

Casings now give a tiny bit of metal so the janitor gets a buff

spacing

DaxDupont commented:
@BeeSting12 it could probably be overwritten to be less painful for the detective's ammo. 
If anyone wants to give me numbers that would be acceptable.

DaxDupont commented:
@vuonojenmustaturska ree now i need to redo parts

Repukan commented:
Makes no sense that ammo you're supposed to use sensibly or spend more tc refilling it is easily available from normal lathes. If you want to make it a worthwhile nerf, just remove it from hacked lathes.

Bombpants commented:
Has it really been abused?

AlexTheSysop42 commented:
>Makes no sense that ammo you're supposed to use sensibly or spend more tc refilling it is easily available from normal lathes. If you want to make it a worthwhile nerf, just remove it from hacked lathes.

It's supposed to be balanced around being really expensive to make, maybe have the nerf only apply to .357 loaders because that's the one that's abused

AlexTheSysop42 commented:
>Has it really been abused?

>After seeing ye olde print infinite ammo murderboning I figured this would be a change that people might get behind.

DaxDupont commented:
@AlexTheSysop42 might make more sense to make detective .38 cheaper instead.

ExcessiveUseOfCobblestone commented:
:( good change but dang that sucks lol

Hatfish commented:
this is a good change, detectives should learn to play their roles.  don't act like you're a cop with ballistics if you don't want to run out of ammo, slick.

ExcessiveUseOfCobblestone commented:
They aren't going to run out of ammo, they're just going to eat up your metal reserves

Mickyan commented:
Should the autolathe even be able to produce lethal ammo? Now that it's public access in most maps it seems overkill, even if you need to hack it first

Spaceinaba commented:
lethal ammo does nothing without a gun so I don't see the issue
we have no productive way to craft our own ammo with gunpowder and metal so for the moment I don't see an issue with having it be in lathes

ExcessiveUseOfCobblestone commented:
@Mickyan ammo doesn't kill people so it's not malicious on it's own anymore than a timer that only goes on grenades and TTVs (maybe circuits?).

Iamgoofball commented:
I used to do this back in the day, good riddance

Anonmare commented:
Metal is ludicrously common anyway

Kromgar commented:
I mean does it matter? One miner can bring back 2,000 metal

KorPhaeron commented:
Conservation of matter

RandomMarine commented:
Extra magazines/speedloaders from uplink only.
Make lathes only make the ammo with no container.

Kromgar commented:
@RandomMarine great compromise

improvedname commented:
>realistic change not being able to refund a speedloader/ammo box for the metal your orginally used

>autistic reaction nobody wants such change because it makes to difficult for them yet they say metal is easy to get yet they dont want it

>pacifist change remove 357. From autolathe

Shadowflame909 commented:
>can make infinite ammo without any cost

Who put them into the autolathe without accounting for this??

ExcessiveUseOfCobblestone commented:
Its not actually 0 cost, you can just get a full refund if you keep the loader

Shadowflame909 commented:
Well thats pretty silly, but I say this PR seems pretty much needed. Everything in ss13 has to have some sort of cost!
